src/cf.data.pre

```diff
@@ -473,6 +473,26 @@ DOC_START
 	CAP_IPC_LOCK capability, or equivalent.
 DOC_END
 
+NAME: hopeless_kid_revival_delay
+COMMENT: time-units
+TYPE: time_t
+LOC: Config.hopelessKidRevivalDelay
+DEFAULT: 60 minutes
```

Comment:
Though you a right that current squid time parsing code is far from ideal and allows such oddities as '60 minute', the current squid default configuration documentation contains lots of such examples, e.g., 'peer_connect_timeout 30 seconds', read_timeout 15 minutes', 'positive_dns_ttl 6 hours' and so on. The current 60 minutes Config.hopelessKidRevivalDelay does not make worse the existing order of things. Probably we need to improve the existing parsing code, but this is out of scope.
Leaving as is for now.